Index: src/main/java/pro/sky/animal_shelter_telegram_bot/model/Animal.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/pro/sky/animal_shelter_telegram_bot/model/Animal.java b/src/main/java/pro/sky/animal_shelter_telegram_bot/model/Pet.java
rename from src/main/java/pro/sky/animal_shelter_telegram_bot/model/Animal.java
rename to src/main/java/pro/sky/animal_shelter_telegram_bot/model/Pet.java
--- a/src/main/java/pro/sky/animal_shelter_telegram_bot/model/Animal.java	(revision 29ada7656e3ffad052124312795aae5b4a4b01d0)
+++ b/src/main/java/pro/sky/animal_shelter_telegram_bot/model/Pet.java	(date 1655925732071)
@@ -1,4 +1,4 @@
 package pro.sky.animal_shelter_telegram_bot.model;
 
-public class Animal {
+public class Pet {
 }
